src/nix-channel/nix-channel.cc

```diff
@@ -165,7 +165,7 @@ static int main_nix_channel(int argc, char ** argv)
     {
         // Figure out the name of the `.nix-channels' file to use
         auto home = getHome();
-        channelsList = settings.useXDGBaseDirectories ? createNixStateDir() + "/channels" : home + "/.nix-channels";
+        channelsList = settings.useXDGBaseDirectories ? createNixStateDir() + OS_STR("/channels") : home + OS_STR("/.nix-channels");
```

Comment:
Was this change necessary for the windows build? Otherwise I think we actually *don't* want this yet because it will break the windows build.

Reply:
I've removed `OS_STR` in https://github.com/NixOS/nix/pull/11281/commits/9ccad9708eac3593defab8a8f74d3acf1bb60870